### PR TITLE
Review

### PR DESCRIPTION
**Pull request by arol**

### CODE REVIEW

File: Middlewares/authorization.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const authMiddleware = async (req, res, next) => {
+  const authHeaders = req.headers.authorization;
+  if (authHeaders) {
+    const [authType, token] = authHeaders.split(' ');
+    if (authType === 'Bearer') {
+      const jwtSecret = process.env.JWT_SECRET;
+      try {
+        const { email } = await jwt.verify(token, jwtSecret);
+        const [user] = await sequelize.query('SELECT * FROM users WHERE email = :email',
```
Comment (arol) [inline Middlewares/authorization.js]: You can actually save user's data in jwt.

File: Models/AdminModels/productModel.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import sequelize from '../../db';
+import CONSTANTS from '../../_CONSTANTS';
+import QUERIES from '../rawqueries';
+
+async function deleteProduct(productId) {
+  sequelize.query(`${QUERIES.deleteProductById}`, {
```
Comment (arol) [inline Models/AdminModels/productModel.js]: `async` without `await`.

Comment (arol) [reply Models/AdminModels/productModel.js]: Also, if you have a function that only runs a promise-based function, just return it and avoid labeling it as `async`.

File: Models/AdminModels/productModel.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    replacements: { productId },
+    type: sequelize.QueryTypes.DELETE,
+  });
+}
+
+async function addProduct(toBeInserted) {
```
Comment (arol) [inline Models/AdminModels/productModel.js]: Unnecessary `async`

File: Models/AdminModels/productModel.js (hunk 2 above)
Comment (ctfrancia) [reply Models/AdminModels/productModel.js]: removed

File: Models/AdminModels/productModel.js (hunk 3 above)
Comment (ctfrancia) [reply Models/AdminModels/productModel.js]: removed

**Pull request closed without merging**